.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,35 @@
+# Description
+
+Please include a summary of the change or the fix you've made. Please also include relevant motivation and context. List any dependencies that are required for this change.
+
+Fixes # (issue)
+
+## Type of change
+
+Please select atleast one
+
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+- [ ] This change requires a documentation update
+
+## Testing
+How was this change tested?
+
+## Documentation
+
+_Please list any documentation changes or additions. Please delete if not necessary._
+
+## Addtional Info
+
+_Please enter any additional information that could be helpful for the reviewers or just to document. Please delete if not necessary._
+
+## Checklist:
+
+- [ ] I have run swiftformat or other relevant tools to follow style guidelines.
```

Comment:
Would be nice if this was automated. I'm not sure we have format rules for this in the repo yet.

Reply:
Yes, we should add linters to CI, we can use something like Danger for this

Reply:
+1 to removing it from Checklist and adding a linter.

Reply:
We've been doing this for kotlin for a while.

Reply:
I am going to leave the scope of this PR to just adding the PR template and create another issue to add linters to CI so it can be tracked separately. Please let me know if anyone disagrees.

Reply:
[Issue](https://github.com/square/workflow/issues/1107) for adding linters to CI.

Reply:
Let's delete this line item for now.